.github/workflows/cdk-bootstrap.yml

```diff
@@ -0,0 +1,92 @@
+name: Bootstrap Geostore AWS accounts with latest version
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  run:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - name: Use Node for CDK deployment
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org
+
+      - name: Cache Node.js packages
+        uses: actions/cache@v2.1.6
+        with:
+          path: ~/.npm
+          key:
+            ${{ runner.os }}-node-${{ secrets.CACHE_SEED }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ secrets.CACHE_SEED }}-
+
+      - name: Install Node.js dependencies
+        run: npm ci --production
+
+      - name: Add local Node packages to PATH
+        run: echo "./node_modules/.bin:$PATH" >> $GITHUB_PATH
+
+      - name: (CI) Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1.5.10
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ap-southeast-2
+          mask-aws-account-id: true
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_CI }}
+
+      - name: Print CDK version
+        run: poetry run cdk --version
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Bootstrap CDK
+        run: |
+          poetry run cdk bootstrap aws://unknown-account/ap-southeast-2
+
+      - name: (Non-Prod) Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1.5.10
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ap-southeast-2
+          mask-aws-account-id: true
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_NON_PROD }}
+
+      - name: Print CDK version
+        run: poetry run cdk --version
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Bootstrap CDK
+        run: |
+          poetry run cdk bootstrap aws://unknown-account/ap-southeast-2
+
+      - name: (Prod) Configure AWS credentials
```

Comment:
Should we maybe make each of the bootstraps a separate job? Then they can run in parallel, and should all end up in the same state rather than a failure in one step resulting in the subsequent environments not being updated at all.